pkg/controller/perconaservermongodb/mgo.go

```diff
@@ -513,22 +522,19 @@ func (r *ReconcilePerconaServerMongoDB) updateConfigMembers(ctx context.Context,
 			continue
 		}
 
-		switch member.State {
-		case mongo.MemberStatePrimary, mongo.MemberStateSecondary, mongo.MemberStateArbiter:
-			membersLive++
-		case mongo.MemberStateStartup,
-			mongo.MemberStateStartup2,
-			mongo.MemberStateRecovering,
-			mongo.MemberStateRollback,
-			mongo.MemberStateDown,
-			mongo.MemberStateUnknown:
+		podName, ok := tags["podName"]
+		if !ok {
+			continue
+		}
```

Comment:
We can write this in a more compact way if we want given that `podName` is not used furthermore, like this:

```
if podName, ok := tags["podName"]; ok {
    rsMembers[podName] = api.ReplsetMemberStatus{
        Name:     member.Name,
        State:    member.State,
        StateStr: member.StateStr,
    }
}

```

it is also more consistent with the style we are using in the same function.

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/1781/commits/1245c27c859eabfb6d009d2407db34bf81a896d1